build.assets/Makefile

```diff
@@ -88,6 +88,35 @@ PULL_ON_CI = $(if $(filter $(CI),true),docker inspect --type=image $(1) >/dev/nu
 # more details.
 RELEASE_TARGET ?= release
 
+# This variable extracts the name of the Docker buildx builder that uses the Docker driver.
```

Comment:
Since this line is so far from the variable, perhaps word it as "DOCKER_BUILD_NAME extracts the name of ..." - that also matches Go style.

Reply:
9c41a6810d1eedbf1a9f170ba4c7e8ed6075c288